VideoLocker/res/layout/row_course_list.xml

```diff
@@ -156,7 +159,8 @@
             android:layout_alignParentRight="true"
             android:layout_alignTop="@id/schoolcode_layout"
             android:layout_below="@id/course_name"
-            android:layout_marginRight="10dp" >
+            android:layout_marginRight="10dp"
+            android:layout_toRightOf="@id/schoolcode_layout">
```

Comment:
@shahidtamboli This is incorrect. The view which is aligned to the right of the parent, should not have <code>toRightOf</code> property.  We should have <code>schoolcode_layout</code> <code>toLeftOf</code> this layout.

So that, if this app runs on a device with bigger width, <code>schoolcode_layout</code> will scale to occupy maximum width. 

@aleffert Though, when there is no space left, we have to either make the text multi-line or ellipsize it (which will happen with <code>maxWidth</code> also). But <code>toLeftOf</code> ensures that the <code>schoolcode_layout</code> takes maximum available width and doesn't overlap.